ui/src/components/home/VisionMissionSectionMobile.js

```diff
@@ -0,0 +1,373 @@
+                      <Text
+                        color="#141419"
+                        fontSize="20px"
+                        fontFamily="DMSerifDisplay_Italic, serif"
+                        fontWeight="400"
+                        lineHeight="28.84px"
+                        wordWrap="break-word"
+                        display="inline"
+                      >
+                        world.
+                      </Text>
+                    </Text>
+                  </Box>
+                </VStack>
+                <VStack>
+                  <VStack
+                    borderRadius="0.5rem"
+                    border="1px solid"
+                    borderColor="green.400"
+                    p="1em"
+                  >
+                    <Box>
+                      <Text
+                        color="#0C0C20"
+                        fontSize="14px"
+                        fontFamily="Manrope"
+                        fontWeight="500"
+                        wordBreak="break-word"
+                        textAlign="center"
+                      >
+                        From January 2024 through December 2029, our church will
+                        journey together to fulfil God’s calling for us through
+                        the Saturate Vision in Hong Kong.
+                      </Text>
+                    </Box>
+                    <Box w="100%" p="1em" bgColor="white">
+                      <Text
+                        color="#4A6EEB"
+                        fontSize="16px"
+                        fontFamily="Manrope"
+                        fontWeight="700"
+                        textTransform="uppercase"
+                        lineHeight="16px"
+                        letterSpacing="4px"
+                        wordBreak="break-word"
+                        textAlign="center"
+                      >
+                        SATURATE VISION
+                      </Text>
+                    </Box>
+                    <Text
+                      color="#0C0C20"
+                      fontSize="20px"
+                      fontFamily="DMSerifDisplay_Italic, serif"
+                      fontWeight="400"
+                      lineHeight="21.26px"
+                      wordBreak="break-word"
+                      textAlign="center"
+                    >
+                      To saturate Hong Kong with the knowledge of God's glory by
+                      living out the Kingdom lifestyle and proclaiming the
+                      Gospel, so that we can make more disciples of all nations,
+                      locally, regionally, and globally for the spread of
+                      Jesus's fame.
+                    </Text>
+                  </VStack>
+                </VStack>
+                <Link href="/about-us" id="homepage-about">
+                  <Button
+                    my="auto"
+                    borderRadius="2.75rem"
+                    w="100%"
```

Comment:
button not actually stretchign to fit the container on mobile

<img width="393" alt="image" src="https://github.com/user-attachments/assets/42bca3ec-7b92-400a-a041-fc820952f278">